context-propagation/pom.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <artifactId>smallrye-opentracing-parent</artifactId>
+    <groupId>io.smallrye</groupId>
+    <version>1.3.3-SNAPSHOT</version>
```

Comment:
Could you please update this? We have released a new version

Reply:
done